### ISSUE TITLE
Migrate internal recommendation notes to new VZE recommendation section

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#9892]
Create CSV template so that Joel S. can transfer internal FRB recommendation and updates about crashes to the new recommendations section in the crash pages. 

CSV template will need to include: 
- Case IDs
- Recommendation
- Update

The "Next Step" column will probably need to be entered manually into the new Updates field in the Recommendation section.

![Screen Shot 2022-08-04 at 8.27.19 AM.png](https://images.zenhubusercontent.com/5d68810b05d7792664be403f/afc64bd9-66b5-48d8-a78b-e05b44fded81)

**Comment by patrickm02L** [comment]
@mateoclarke created this Issue from the VZ Sprint Review meeting 8/2/22. Please add any technical criteria missing.

**Comment by patrickm02L** [comment]
Ask Joel S. to provide us what data he has rather than creating a template.

**Comment by patrickm02L** [comment]
This is one way to do this. 
https://www.postgresqltutorial.com/postgresql-tutorial/import-csv-file-into-posgresql-table/

**Comment by patrickm02L** [comment]
Pretty well groomed. Dependent on #9519.

**Comment by mateoclarke** [comment]
#9519 is now merged into production and this should be good to assign in the next sprint.

**Comment by patrickm02L** [comment]
@mtvo-tx asked that we having a meeting with her and Joel M. after we've put together the script that will migrate the Recommendations and notes.

**Comment by patrickm02L** [comment]
In Sprint Review 9/12/22 Joel asked that we include the Coordinating Partners that are a single value. The ones will multiple values will need to be addressed in another Issue.

**Comment by patrickm02L** [comment]
#10193 has a link to the spreadsheet data we want to import.

**Comment by chiaberry** [comment]
@patrickm02L one of the required fields for the recommendations table is "created_by". 

I was leaning towards making it be "import script", but noticed current records have it as an email address. Should it be Joel's email? or someone elses

**Comment by patrickm02L** [comment]
Let's put Joel's email, he's the one that maintains the spreadsheet, or did until Mary got here. I think she's taking over the FRB meetings going forward.

**Comment by chiaberry** [comment]
ran migration this morning, updated Joel and Mary.
